components/draw/GeometryMode.svelte

```diff
@@ -0,0 +1,156 @@
+<script>
```

Comment:
This was "Edit mode" yesterday. "Edit attributes" and "edit geometry" sound OK, or any other ideas?